src/SFA.DAS.Assessor.Functions.ExternalApis/Assessor/Constants/CertificateStatus.cs

```diff
@@ -15,5 +15,7 @@ public static class CertificateStatus
         public const string Deleted = "Deleted";
         public const string Reprint = "Reprint";
         public const string Error = "Error";
+        public const string Delivered = "Delivered";
```

Comment:
If you look at CertificateStatus in both Assessor and Admin - they are currently the same duplicated code - this should be another duplicate of it - and you'll set that in Assessor and Admin it has the style of sets of status's as well which you can use

Reply:
Copied from Assessor service